schema.graphql

```diff
@@ -1,12 +1,62 @@
+scalar Date
+
 type Group {
   id: Int
   name: String
 }
 
+input GroupInput {
+  name: String
+}
+
 type Query {
   listGroups: [Group]!
+  listShipments: [Shipment]!
```

Comment:
```suggestion
  listShipments: [Shipment!]!
```

otherwise `[null]` is allowed.

Reply:
Yup, good call. Will address this.

- [ ] TODO: Make `listShipments` return `[Shipment!]!`